src/client/Microsoft.Identity.Client/WwwAuthenticateParameters.cs

```diff
@@ -60,6 +60,13 @@ public IEnumerable<string> Scopes
         /// </summary>
         public string Authority { get; set; }
 
+        /// <summary>
+        /// AAD tenant ID to which the Azure subscription belongs to.
```

Comment:
This can work without a subscription ...

```suggestion
        /// Azure AD tenant ID.
```